cdap-ui/Gulpfile.js

```diff
@@ -289,6 +288,7 @@ gulp.task('watch:js:app:babel', function() {
     .pipe(plug.babel({
       presets: ['es2015']
     }))
+    .pipe(plug.ngAnnotate())
```

Comment:
Hhmm I think this issue addresses it. https://github.com/olov/ng-annotate/issues/245 I guess the developer is not planning on supporting ngAnnotate to run on es6 code.